apps/storefront/src/hooks/dom/utils.ts

```diff
@@ -270,7 +272,11 @@ const addProductsFromCartToQuote = (setOpenPage: SetOpenPage) => {
   };
 };
 
-const addProductFromProductPageToQuote = (setOpenPage: SetOpenPage) => {
+const addProductFromProductPageToQuote = (
+  setOpenPage: SetOpenPage,
+  isEnableProduct: boolean | undefined,
```

Comment:
I think the init value of `isEnableProduct` is false, why do you need to add `undefined` here?